ClangAutoComplete.py

```diff
@@ -11,53 +11,53 @@ class ClangAutoComplete(sublime_plugin.EventListener):
 	compl_regex = re.compile("COMPLETION: ([^ ]+) : ([^\\n]+)")
 	file_ext = re.compile("[^\.]+\.([^\\n]+)")
 	syntax_regex = re.compile("\/([^\/]+)\.(?:tmLanguage|sublime-syntax)")
-	project_name_regex = re.compile("([^\.]+).sublime-project")
 	settings_time = 0
 
+	def to_bool(self, str_flag):
+		return str_flag == "true"
+
 	def load_settings(self):
 		# Only load the settings if they have changed
-		settings_modified_time = path.getmtime(sublime.packages_path()+"/ClangAutoComplete/"+"ClangAutoComplete.sublime-settings")
+		settings_modified_time = path.getmtime(
```

Comment:
You are right, that would be best however I'm not exactly sure how to do this... the sublime API is not always that clear :/. Last I tried, it did not automatically reload the plugin on settings change and that is why I had put that line there.